R/bootstrap.R

```diff
@@ -1342,13 +1343,15 @@ uiOutput <- htmlOutput
 #' @export
 downloadButton <- function(outputId,
                            label="Download",
-                           class=NULL, ...) {
+                           class=NULL,
+                           ..., 
+                          icon=icon("download")) {
```

Comment:
Small nit: can you add one more space, so that this aligns this with the previous lines?

Also, the .Rd file needs to be re-generated. Otherwise, looks good!

Reply:
Done via last commit ✅